packages/core/src/accordion/index.tsx

```diff
@@ -27,3 +27,4 @@ export type {
   AccordionTriggerProps,
 };
 export { Content, Header, Item, Root, Trigger };
+export default { Content, Header, Item, Root, Trigger };
```

Comment:
Exposing default exports is not necessary for this PR. The `package.json` change should be enough.